ci/tasks/check-canaries-in-error-state.yml

```diff
@@ -0,0 +1,47 @@
+---
+platform: linux
+image_resource:
+  type: registry-image
+  source:
+    repository: govukpay/concourse-runner
+params:
+  AWS_ACCESS_KEY_ID:
+  AWS_SECRET_ACCESS_KEY:
+  AWS_SESSION_TOKEN:    
+  AWS_REGION: eu-west-1
+  # ENVIRONMENT is one of "test", "stag" or "prod" as that's what the canaries are appended with (due to the character limitation)."
+  ENVIRONMENT:
+run:
+  path: /bin/bash
+  args:
+    - -ec
+    - |
+      NON_SCHEDULED_CANARIES="notifcatns_sndbx pymntlnk_sandbox cancel_sandbox card_wpay_3ds2ex \
```

Comment:
Could we call them `POST_DEPLOY_CANARIES` instead?

Reply:
Done